NosTests/Controller/NoteEditorControllerTests.swift

```diff
@@ -0,0 +1,218 @@
+import XCTest
+import UIKit
+
+// swiftlint:disable implicitly_unwrapped_optional
+final class NoteEditorControllerTests: CoreDataTestCase {
+    
+    var subject: NoteEditorController!
+    var textView: UITextView!
+
+    override func setUpWithError() throws {
+        subject = NoteEditorController()
+        textView = UITextView()
+        subject.textView = textView
+    }
+    
+    // MARK: - Show Mentions Search
+    
+    func testShowMentionsSearch_whenTypingAfterSpace_thenMentionsSearchIsShown() throws {
```

Comment:
```suggestion
    func testShowMentionsAutocomplete_whenTypingAfterSpace_thenMentionsSearchIsShown() throws {
```
maybe? and if so, ditto below